problem/game.cc

```diff
@@ -0,0 +1,83 @@
+#include "game.h"
+
+#include <iostream>
+
+bool PlayerWonCheck(std::vector<std::vector<std::string>> grid, int player) {
+  // Implement your code here to identify what streak the current player won on
+  // the grid. 
```

Comment:
2D Vector "grid" and remember to properly print the correct streak and return true when needed